changelog.txt

```diff
@@ -1,6 +1,7 @@
 Changes in 6.2.2-preview :
 
 - All: Fixed bug with CoreCLR UserAgent string.
+- Blobs: Fixed boundary checks for lease durations and break lease period to be in line with REST endpoints
```

Comment:
Sorry I didn't catch this before. You should create a "Changes in 7.0.0 :" header and put this change under that since this is a breaking change and it will go out in our 7.0 release